routes/index.js

```diff
@@ -195,6 +195,42 @@ module.exports = function (passport) {
 
   });
 
+
+  router.get('/userManagement/:badge', function(req,res){
```

Comment:
Could you also just put the `checkAuth` middleware function here so only admins can access this route?